nixos/modules/programs/thunderbird.nix

```diff
@@ -0,0 +1,97 @@
+{
+  pkgs,
+  config,
+  lib,
+  ...
+}:
+let
+  cfg = config.programs.thunderbird;
+  policyFormat = pkgs.formats.json { };
+  thunderbirdVersion = "115";
```

Comment:
The issue is that the documentation for the policies do not match the Thunderbird version 1:1. [Here](https://github.com/thunderbird/policy-templates/tree/master/templates/) they have a folder for esr102 esr115 est68 esr78 esr91 and central (the most up to date one).

I could just set the link to point to the most [recent documentation](https://github.com/thunderbird/policy-templates/tree/master/templates/central) at a higher risk that policies are misdocumented

Reply:
Could we ask upstream to make a better QA for this so that the downstream doesn't have to do management like this?

Reply:
They do not have issues activated in their repository and I doubt they would make changes to accommodate us.

The Firefox package links to the most up to date version of the documentation regardless of Firefox version, so I suppose we could just link to the [github repository root](https://github.com/thunderbird/policy-templates) where a user can select their version or to [enterprise.thunderbird.net/manage-updates-policies-and-customization](https://enterprise.thunderbird.net/manage-updates-policies-and-customization/managing-thunderbird-policies/customizing-thunderbird-using-policies.json), which in turn links to GitHub.

I'll write them a support ticket making this request, but this will take some time to happen, if at all.

Reply:
> I doubt they would make changes to accommodate us. -- @Nydragon(https://github.com/NixOS/nixpkgs/pull/311669#discussion_r1601695988)

This rather seems as release management issue rather than NixOS-specific thing to me, thus why i think it might be a good idea to point that out to them.

> The Firefox package links to the most up to date version of the documentation regardless of Firefox version, so I suppose we could just link to the [github repository root](https://github.com/thunderbird/policy-templates) where a user can select their version or to [enterprise.thunderbird.net/manage-updates-policies-and-customization](https://enterprise.thunderbird.net/manage-updates-policies-and-customization/managing-thunderbird-policies/customizing-thunderbird-using-policies.json), which in turn links to GitHub. -- @Nydragon (https://github.com/NixOS/nixpkgs/pull/311669#discussion_r1601695988)

My point is that the less things we need to constantly change on the packages the better as then the development time gets progressively better to focus on other issues and without introducing confusion to the end-user who might experience a situation where they get a wrong version of the documentation.

> I'll write them a support ticket making this request, but this will take some time to happen, if at all. -- @Nydragon (https://github.com/NixOS/nixpkgs/pull/311669#discussion_r1601695988)

Reference the reply if you get any plz might be also a good idea to reference this in the code itself

Reply:
I reached out on [Matrix](https://matrix.to/#/!fubIsJzeAcCcjYTQvm:mozilla.org/$4byxRrFJmBIJRIm9azPT-jVF6HmSjBCNkaetM5Qip2E?via=kewis.ch&via=mozilla.org&via=matrix.org) and their [support site](https://support.mozilla.org/en-US/questions/1447457), i'll update here once I get a reply.

> the less things we need to constantly change on the packages the better

I fully agree, but wouldn't it make sense to change the documentation URL in a subsequent PR instead of blocking this one?

Reply:
Didn't really amount to much, you can look at the Matrix conversation for details.

So we could either point to the repo root, which links to the different ESR versions of the doc or write a small script that selects the previous ESR version based on the packaged Thunderbird version and builds the link.

I think linking to the repo root might be more reliable